src/pds/registrysweepers/utils/misc.py

```diff
@@ -0,0 +1,33 @@
+import random
+from datetime import datetime
+from typing import Any
+from typing import List
+
+
+def coerce_list_type(db_value: Any) -> List[Any]:
+    """
+    Coerce a non-array-typed legacy db record into a list containing itself as the only element, or return the
+    original argument if it is already an array (list).  This is sometimes necessary to support legacy db records which
+    did not wrap singleton properties in an enclosing array.
+    """
+
+    return (
+        db_value
+        if type(db_value) is list
+        else [
+            db_value,
+        ]
+    )
+
+
+def get_human_readable_elapsed_since(begin: datetime) -> str:
+    elapsed_seconds = (datetime.now() - begin).total_seconds()
```

Comment:
There's a [humanize](https://pypi.org/project/humanize/) package that does stuff like this too—but this is pretty concise, which is nice. Just what you need!